packages/preprocess/src/utils/get-route-hash.cjs

```diff
@@ -0,0 +1,19 @@
+const md5 = require("blueimp-md5");
```

Comment:
I was wondering if the default `crypto` module had a `md5`? 🤔

Reply:
It likely does; this is behavior that already existed, so I left it as is. We can attempt to change it if we think it's in scope, but I would lean towards filing an issue for it.

Reply:
Let's keep it for now, and have a code-debt for that